ecs-cli/modules/cli/local/localproject/project.go

```diff
@@ -268,27 +268,39 @@ func (p *LocalProject) Write() error {
 	if err := writeFile(p.LocalOutFileName(), p.composeBytes[baseComposeIndex]); err != nil {
 		return err
 	}
-	logrus.Infof("Successfully wrote %s", p.LocalOutFileName())
 
-	if err := writeFile(p.OverrideFileName(), p.composeBytes[overrideComposeIndex]); err != nil {
+	if err := writeFileIfNotExist(p.OverrideFileName(), p.composeBytes[overrideComposeIndex]); err != nil {
```

Comment:
Didn't we have something like this before where we decided maybe `writeNewFile` might be a more concise naming?

Reply:
oh yeah, it was `createNewComposeProject`. Changed to `writeNewFile`! 

Personally, I prefer `writeFileIfNotExist` because it's a bit more explicit than `writeNewFile` the New doesn't have the same amount of weight it seems.